appium/webdriver/webdriver.py

```diff
@@ -228,8 +228,11 @@ def __init__(
             AppiumConnection.set_certificate_bundle_path(None)
             urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
+        if isinstance(command_executor, (str, bytes)):
```

Comment:
can it be `bytes`?

Reply:
Added bytes as selenium had, but hm, maybe my reference was old one. Latest selenium is str